parallel-crypto3/parallel-crypto3.nix

```diff
@@ -0,0 +1,43 @@
+{ lib,
+  stdenv,
+  ninja,
+  pkg-config,
+  cmake,
+  boost,
+  gdb,
+  cmake_modules,
```

Comment:
why we need it?

Reply:
Do you mean cmake_modules? it is used in parallel-crypto3 the same as in the original crypto3

Reply:
got it, seem i can't to resolve threads